test/videoframeprocessor/NoOpVideoFrameProcessor.test.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as chai from 'chai';
+import { expect } from 'chai';
+
+import CanvasVideoFrameBuffer from '../../src/videoframeprocessor/CanvasVideoFrameBuffer';
+import NoOpVideoFrameProcessor from '../../src/videoframeprocessor/NoOpVideoFrameProcessor';
+import VideoFrameBuffer from '../../src/videoframeprocessor/VideoFrameBuffer';
+
+describe('NoOpVideoFrameProcessor', () => {
+  const assert: Chai.AssertStatic = chai.assert;
+  let proc: NoOpVideoFrameProcessor;
+  beforeEach(() => {
+    proc = new NoOpVideoFrameProcessor();
+  });
+  describe('construction', () => {
+    it('can be constructed', () => {
+      assert.exists(proc);
+    });
+  });
+
+  describe('process', () => {
+    it('passes through the buffers', async () => {
+      const buffers: VideoFrameBuffer[] = [];
+      let output = await proc.process(buffers);
+      expect(output).to.deep.equal(buffers);
+      buffers.push(new CanvasVideoFrameBuffer(null));
+      output = await proc.process(buffers);
+      expect(output).to.deep.equal(buffers);
```

Comment:
I *think* you should be using `to.be` or `to.eql` here — `NoOpVideoFrameProcessor` passes the exact input to the output, right?

Reply:
yes, changed